dashboard/src/shared/utils.test.ts

```diff
@@ -0,0 +1,158 @@
+// Copyright 2022 the Kubeapps contributors.
```

Comment:
Adding a test for the whole utils.ts file as we didn't have any yet.

Reply:
Excellent, that'll encourage more tests :)